js/i18n.js

```diff
@@ -239,18 +244,17 @@ function languageSelectorTimeout(textBox) {
   var $textBox = $(textBox);
   var textVal = $textBox.val();
 
-  // don't search for nothing
-  if (textVal.length < 2) return;
-
   var lastSearch = $textBox.attr('data-last-search');
 
   // limit the search to the first 4 characters
   var thisSearch = (textVal.length > 4) ? textVal.substr(0, 4) : textVal;
+  $textBox.attr('data-last-search', thisSearch);
```

Comment:
Just to explain this change: It was failing to show any languages if you:

1) Type 'en' in the search (List with "English (en)" at the top appears)
2) Backspace so it just has 'e' (list goes away)
3) Type'n' again...no list.

We always need to track what was last looked at, storing in data-last-search even if we didn't actually search. Fixed this check in the test as well.